dom/shopping-list.html

```diff
@@ -1,38 +1,48 @@
 <!-- source code available : https://github.com/mdn/learning-area/blob/master/javascript/apis/document-manipulation/shopping-list.html -->
 
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8">
-    <title>Shopping list example</title>
-    <style>
-      li {
-        margin-bottom: 10px;
-      }
-
-      li button {
-        font-size: 8px;
-        margin-left: 20px;
-        color: #666;
-      }
-    </style>
-  </head>
-  <body>
-
-    <h1>My shopping list</h1>
-
-    <div>
-      <label for="item">Enter a new item:</label>
-      <input type="text" name="item" id="item">
-      <button>Add item</button>
-    </div>
-
-    <ul>
-
-    </ul>
-
-    <script>
-
-    </script>
-  </body>
+	<html>
+<head>
+<meta charset="utf-8">
+<title>Shopping list example</title>
+<link rel="stylesheet" type="text/css" href="shopping-list.css">
+</head>
```

Comment:
In HTML, it is convention to tab over for everything "inside" a tag, or between an open tag <> and a closing tag </>. That's something you'll want to get used to doing, and applies to your whole HTML doc.